src/api/admin.ts

```diff
@@ -518,6 +518,22 @@ export const removeVenueOwner = async (venueId: string, ownerId: string) =>
     ownerId,
   });
 
+export const addSpaceOwnerBulk = async (
+  addedSpacesIds: string[],
```

Comment:
Can these be `SpaceId[]`, `UserId`, `WorldId` types or is there too much of an effort to stop TS from crying?